src/osbuild-manifests/aws_tag.py

```diff
@@ -0,0 +1,79 @@
+import os
+import subprocess
+import json
+import argparse
+
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--stream', dest='stream', type=str, help='Fedora stream', required=True)
+    args = parser.parse_args()
+
+    builds = getBuildsForStream(args.stream)
+    for build in builds:
+        build_id=build['id']
+        arches=build['arches']
+        for arch in arches:
+            print(f"The build is {build_id}")
+            buildFetch(args.stream, build_id, arch)
+            meta = open('builds/'+build_id+'/'+arch+'/meta.json')
+            data = json.load(meta)
+
+            # Delete this when actually running. Just here while I make this script
+            data ={"amis":[{
+                "name": "us-east-1",
+                "hvm": "ami-0016d5df3041499f9",
+                "snapshot": "snap-0c1ca4850fcd5e573"
+            }]}
+            amis = data['amis']

```

Comment:
we need to make sure we handle the case when there is no ami in the `meta.json`. This can happen for multi-arch  manifests that don't have AMIs (i.e. ppc64le and s390x) or builds that failed half way through but had uploaded the meta.json for the ostree.